### PR TITLE
gopy: add a CONTRIBUTE guide

### PR DESCRIPTION
**Pull request by sbinet**
Fixes go-python/gopy#90.

### CODE REVIEW

**Comment by corona10** [comment]
@sbinet  Looks great. I will translate it when this guide is merged.
Just specifying the directory where CONTRIBUTE_KR.md would landed.

**Comment by sbinet** [comment]
SGTM

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+The `go-python` project (and `gopy`) eagerly accepts contributions from the community.
+
+## Introduction
+
+
+The `go-python` project provides libraries and tools in Go for the Go community to better mesh with Python projects and libraries, and we would like you to join us in improving `go-python`'s quality and scope.
```
Comment (btracey) [inline CONTRIBUTE.md]: integrate/interact/interoperate instead of mesh?

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+This document is for anyone who is contributing or interested in contributing.
```
Comment (btracey) [inline CONTRIBUTE.md]: is for contributors or those interested in contributing

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+Questions about `go-python` or the use of its libraries can be directed to the [go-python](mailto:go-python@googlegroups.com) mailing list.
```
Comment (btracey) [inline CONTRIBUTE.md]: not sure why, but "and" instead of "or" feels much more common here.

File: CONTRIBUTE.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+- Avoid destructive behavior
+
+Excerpted from the [Go conduct document](https://golang.org/conduct).
+
+### Reporting Bugs
+
+When you encounter a bug, please open an issue on the corresponding repository.
+Start the issue title with the repository/sub-repository name, like `bind: issue name`.
+Be specific about the environment you encountered the bug in (_e.g.:_ operating system, Go compiler version, ...).
+If you are able to write a test that reproduces the bug, please include it in the issue.
```
Comment (btracey) [inline CONTRIBUTE.md]: I don't like the dual use of "test" in this bullet and the next. It's likely to be somewhat confusing for newer users.

"If you are able to write a reproducer for the bug, please include it in the issue"

File: CONTRIBUTE.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+For exceptions, added code must also follow a BSD license.
+
+### Code Contribution
+
+If it is possible to split a large pull request into two or more smaller pull requests, please try to do so.
+Pull requests should include tests for any new code before merging.
+It is ok to start a pull request on partially implemented code to get feedback, and see if your approach to a problem is sound.
+You don't need to have tests, or even have code that compiles to open a pull request, although both will be needed before merge.
+When tests use magic numbers, please include a comment explaining the source of the number.
+Benchmarks are optional for new features, but if you are submitting a pull request justified by performance improvement, you will need benchmarks to measure the impact of your change, and the pull request should include a report from [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp) or, preferably, [benchstat](https://github.com/rsc/benchstat).
```
Comment (btracey) [inline CONTRIBUTE.md]: benchstat is in perf: golang.org/x/perf/cmd/benchstat

File: CONTRIBUTE.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+Please always format your code with [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports).
+Best is to have it invoked as a hook when you save your `.go` files.
+
+Files in the `go-python` repository don't list author names, both to avoid clutter and to avoid having to keep the lists up to date.
+Instead, your name will appear in the change log and in the [CONTRIBUTORS](https://github.com/go-python/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/go-python/license/blob/master/AUTHORS) files.
+
+New files that you contribute should use the standard copyright header:
+
+```
+// Copyright 2017 The go-python Authors.  All rights reserved.
```
Comment (btracey) [inline CONTRIBUTE.md]: You may want to have 20xx or YEAR

File: CONTRIBUTE.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+  * ` LGTM ` — looks good to me
+  * ` SGTM ` — sounds good to me
+  * ` CL ` — change list; a single commit in the repository
+  * ` s/foo/bar/ ` — please replace ` foo ` with ` bar `; this is [sed syntax](http://en.wikipedia.org/wiki/Sed#Usage)
+  * ` s/foo/bar/g ` — please replace ` foo ` with ` bar ` throughout your entire change
+
+We follow the convention of requiring at least 1 reviewer to say LGTM before a merge.
+When code is tricky or controversial, submitters and reviewers can request additional review from others and more LGTMs before merge.
+You can ask for more review by saying PTAL in a comment in a pull request.
```
Comment (btracey) [inline CONTRIBUTE.md]: You probably want PTAL in the list above.

File: CONTRIBUTE.md (hunk 1 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 2 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 3 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 4 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 5 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 6 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

File: CONTRIBUTE.md (hunk 7 above)
Comment (sbinet) [inline CONTRIBUTE.md]: done

**Pull request merged**